src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala

```diff
@@ -0,0 +1,23 @@
+package akka.persistence.cassandra.query
+
+import scala.concurrent.duration._
+import com.typesafe.config.Config
+import com.datastax.driver.core.ConsistencyLevel
+import akka.persistence.cassandra.journal.CassandraJournalConfig
+
+private[query] class CassandraReadJournalConfig(config: Config, writePluginConfig: CassandraJournalConfig) {
```

Comment:
I still do not think the read journal config should have dependency on cassandra journal config. The client than may need to configure tthings they actually don't need.